db/langCach_vec.py

```
@@ -0,0 +1,47 @@
+from config import LANGCACHE_API_KEY, LANGCACHE_ID, LANGCACHE_URL
+
+class LangCacheClient:
+    def __init__(self):
+        try:
+            self.lang_cache = LangCache(
+                server_url=LANGCACHE_URL,
+                cache_id=LANGCACHE_ID,
+                api_key=LANGCACHE_API_KEY
+            )
+            print("✅ Connected to LangCache")
+        except Exception as e:
```

Ruff (0.15.12)

[warning] Do not catch blind exception: `Exception`

(BLE001)